kbase-extension/static/kbase/js/kbaseNarrative.js

```diff
@@ -778,24 +780,20 @@ define([
                     this.sidePanel.render();
                 }.bind(this));
 
-            $([Jupyter.events]).trigger('loaded.Narrative');
             $([Jupyter.events]).on('kernel_ready.Kernel',
-                function () {
-                    console.log('Kernel Ready! Initializing Job Channel...');
+                (e) => {
                     this.loadingWidget.updateProgress('kernel', true);
+                    this.jobCommChannel = new JobCommChannel();
                     // TODO: This should be an event "kernel-ready", perhaps broadcast
                     // on the default bus channel.
-                    this.sidePanel.$jobsWidget.initCommChannel()
```

Comment:
This widget is removed... Was the widget even viewable before?

Reply:
Nope. Once upon a time, there was a separate "Jobs" tab in the Narrative interface. That widget drove it. Then that got removed, and it wasn't visible anymore, but the widget was still needed for kernel communication.

Most of this PR is removing all of that old mess and turning the "Job Panel" widget just into a communication channel.